src/devtools/global.styl

```diff
@@ -95,3 +95,17 @@ $arrow-color = #444
     text-align center
     padding 0.5em
     margin 0 auto
+
+@-moz-keyframes rotate
+  0%
+    -moz-transform rotate(0deg)
```

Comment:
No need for prefixes because stylus will add them. So keeping the `@keyframes` rule with only `transform` is enough